src/containers/createArticle.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import styles from '../styles/articleStyles/NewArticle.scss';
+import {createArticleAction, editArticle} from '../actions/articleActions/articleActions';
+import {editOneArticleActionCreator} from '../actions/articleActions/articleActionCreators';
+import {Bottom, EditorCreate, Header} from '../components/articles/editorCreate';
+
+class CreateArticle extends React.Component {
+
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			title: '', titleError: '', description: '',
+			descriptionError: '', body: '', bodyError: ''
+		};
+
+		this.handleSubmit = this.handleSubmit.bind(this);
+		this.handleChange = this.handleChange.bind(this);
```

Comment:
`handleChange`  is an arrow function so binding it in the constructor is unnecessary.

Reply:
This has been resolved @collinewait, feel free to check it out.